public/app/controls-test.js

```
@@ -0,0 +1,45 @@
+    }
+
+    // Play / transport controls
+    check('playBtn exists', document.getElementById('playBtn'));
+    check('tpPlay exists', document.getElementById('tpPlay'));
+    check('clearFile exists', document.getElementById('clearFile'));
+    check('fileInput exists', document.getElementById('fileInput'));
+
+    // Slider tabs
+    var tabs = ['gate', 'nr', 'eq', 'dyn', 'spec', 'adv', 'sep', 'out'];
+    tabs.forEach(function (t) {
+      check('tab-' + t + ' exists', document.getElementById('tab-' + t));
```

ESLint

[error] 'document' is not defined.

(no-undef)